.gitmodules

```diff
@@ -19,4 +19,4 @@
 [submodule "windows_docker_resources/ros2-cookbooks"]
 	path = windows_docker_resources/ros2-cookbooks
 	url = git@github.com:ros-infrastructure/ros2-cookbooks
-	branch = latest
+	branch = brawner/update-openssl
```

Comment:
I guess this should remain as `latest`, assuming https://github.com/ros-infrastructure/ros2-cookbooks/pull/15 is merged. Right?

Reply:
Right, I'll update this PR when the ros2-cookbooks is merged. Still waiting for the CI job to complete